### ISSUE TITLE
perm check not working

### ISSUE THREAD

**Issue by JohnKillswitch** [Arasple/Invero#5]
Seems like perm check not working

have this code

https://pastebin.com/waAjGcn4

i havent a map.hidden permission but anyway action is going to else instead of then

**Comment by Flaimer2** [comment]
Я внес изменения в ваше меню: https://pastes.dev/XWQ5TKkPKp Вы забыли добавить команду с разрешением "inline", которое необходимо для использования "kether" в строке. Если не добавлять "inline", то можно использовать заполнитель "%player_name%". И кстати вы в else забыли "|-"

**Comment by JohnKillswitch** [comment]
Все равно не работает
https://pastes.dev/nHe6zSPg1Q

Вот ошибка:
https://pastes.dev/omZBa957sn

**Comment by Flaimer2** [comment]
Какая у вас версия?

**Comment by JohnKillswitch** [comment]
Сегодня из сурсоу сбилдил

вт, 4 апр. 2023 г., 00:56 Flaimer2 ***@***.***>:

**Comment by Flaimer2** [comment]
Попробуйте сбилдить от сюда: https://gitlab.com/Arasple/Invero

**Comment by JohnKillswitch** [comment]
Так, оно вроде бы работает. Но проблема в том, что игрок не может переключать этот пермишенс сразу. Ему нужно перезаходить в меню, чтобы это работало

> Попробуйте использовать: command inline "команда" as console

**Comment by Flaimer2** [comment]
https://pastes.dev/JzC6CJMcPa
